pkg/prowgen/prowgen.go

```diff
@@ -394,7 +394,7 @@ func AlwaysRunInjector() JobConfigInjector {
 					// Prevent sneaking in wrong settings from previous runs of "make jobs".
 					// Make sure we reset it to default.
 					jobConfig.PresubmitsStatic[k][i].AlwaysRun = false
-					jobConfig.PresubmitsStatic[k][i].SkipIfOnlyChanged = "^.tekton/.*|^.konflux.*|^.github/.*"
+					jobConfig.PresubmitsStatic[k][i].SkipIfOnlyChanged = "^.tekton/.*|^.konflux.*|^.github/.*|^rpms.lock.yaml$|^hack/.*|^OWNERS.*|.*\\.md"
```

Comment:
maybe include the `.konflux-release` folder from SO with the override-snapshots too

Reply:
yes, it should covered